guides/release-process.md

```diff
@@ -15,6 +15,21 @@ The `@cypress/`-namespaced NPM packages that live inside the [`/npm`](../npm) di
   - Permissions for your npm account to publish the `cypress` package.
   - Permissions to update releases in ZenHub.
 
+- [Set up](https://cypress-io.atlassian.net/wiki/spaces/INFRA/pages/1534853121/AWS+SSO+Cypress) an AWS SSO profile with the [Team-CypressApp-Prod](https://cypress-io.atlassian.net/wiki/spaces/INFRA/pages/1534853121/AWS+SSO+Cypress#Team-CypressApp-Prod) role. The release scripts assumes the name of your profile is `prod`. Make sure to open the "App Developer" expando for some necessary config values. You AWS config file should end up looking like the following:
+
+    ```
```

Comment:
Mine doesn't have all of these configured and it works as expected. IMO we shouldn't add this since the Wiki pages provides it and the infra team will make updates when needed.

Reply:
Would it be better to update the links & wiki to make it clear what was missing?